dynamicio/core.py

```diff
@@ -263,8 +263,11 @@ def _has_valid_dtypes(self, df: pd.DataFrame) -> bool:
             bool - `True` if `df` has the given dtypes, `False` otherwise
         """
         dtypes = df.dtypes
+        print("dtypes: ", dtypes)
+        print("\ndf:", df)
 
         for column_name, expected_dtype in self.schema.items():
+            print(f"\n{column_name} ---- {expected_dtype}")
```

Comment:
I assume these were used for debugging, and we want to remove these print statements?

Reply:
yeah sorry about that.
I added that when i was debugging the dtypes. I will remove them

Reply:
Thanks @KitBurgess I missed them and merged had to force-push a fix earlier today. It's all good now guys.

Reply:
Great! thanks.
Then no need resolve it on my end